src/ioc.config.ts

```diff
@@ -31,8 +38,8 @@ function registerAll(): void {
 
 export const Controller = {
 	container,
-	reset: (): void => {
-		container.reset();
+
+	registerAll: (): void => {
```

Comment:
I was originally trying to pass the class type here to register. But since `src` cannot view contents of `test` (without change to `tsconfig` - adding that may cause cyclic dependency), I decided to move the registering portion out of here as well.